content/applications/websites/ecommerce/managing_products/multi_images.rst

```diff
@@ -1,19 +1,24 @@
-=========================================
-How to display several images per product
-=========================================
+===========================
+Multiple images per product
+===========================
 
-By default your product web page displays the main image of your product only.
-If you like to show your products under several angles, you can turn the image into a carrousel.
+When you register several images for a product, only the first image is shown on the main products
+page. By clicking on the product, visitors can see your products from several angles with the
+carousel view.
```

Comment:
why "angles"? It can be simply that you have several images (presentations, details, etc) let's not assume it's only about angles :D 

Please also keep in mind that from Odoo 16 you have also the grid view in addition to the carousel view. So maybe it's best to change the intro :)

Reply:
Ok, I will update the intro for V16 as well as explain the front-end process.